docs/balancing.md

```diff
@@ -27,34 +27,33 @@ Therefore, any `Duration` object with nonzero years or months can refer to a dif
 No balancing is ever performed between years, months, and days, because such conversion would be ambiguous.
 If you need such a conversion, you must implement it yourself, since the rules can depend on the start date and the calendar in use.
 
-Negative values are never allowed as `Temporal.Duration` fields, so passing one as an argument to `new Temporal.Duration()` or as a property in the object passed to `Temporal.Duration.from()` will always throw an exception, regardless of the disambiguation mode.
+`Temporal.Duration` fields are not allowed to have mixed signs, so, for example, passing one positive and one negative argument to `new Temporal.Duration()` or as a properties in the object passed to `Temporal.Duration.from()` will always throw an exception, regardless of the disambiguation mode.
 
 Therefore, the only case where constrain and reject mode have any effect when creating a duration, is integer overflow.
-If one of the values overflows, constrain mode will cap it to `Number.MAX_VALUE`.
+If one of the values overflows, constrain mode will cap it to `Number.MAX_VALUE` or `-Number.MAX_VALUE`.
 
 ## Duration arithmetic
 
-When adding two `Temporal.Duration`s, the situation is much the same as for constructing one.
-It's not possible to end up with a negative value when adding two valid durations, nor is any balancing needed, so the disambiguation mode only determines what to do in the case of integer overflow.
+When adding or subtracting two `Temporal.Duration`s, the situation is different.
```

Comment:
This wording is confusing now. Instead of "When adding or subtracting two `Temporal.Duration`s", how about something like this:  

> When adding durations of opposite signs (or subtracting durations of the same sign)